### ISSUE TITLE
improve lotus generate documentation

### ISSUE THREAD

**Issue by agenteo** [lotus/lotus#271]
Currently `lotus` helper message says: `lotus generate         # generates a new action` which I reckon should be changed to `run code generator` or perhaps removed.

When typing `lotus generate` I currently see:

```
Usage:
  lotus generate

Options:
      [--application-base-url=APPLICATION_BASE_URL]  # application base url
      [--path=PATH]                                  # applications path
                                                     # Default: apps
      [--skip-view], [--no-skip-view]                # skip the creation of view and templates (only for action)
  -h, [--help=HELP]                                  # displays the usage method

generates a new action
```

I think this should be updated with the latest options. I run in to this looking for `lotus generate app`.

**Comment by joneslee85** [comment]
@agenteo thank for the bug reporting. What do you think about submitting a PR for this?

**Comment by jodosha** [comment]
@agenteo We're aware of the fact that it s**ks. Sorry for the poor experience. It requires a refactoring to change the code from the actual state to something similar `db` commands. See https://github.com/lotus/lotus/blob/master/lib/lotus/commands/db.rb

I've tried to start this, but it's time consuming and the v0.4.0 release was close, so I had to abandon it. I can share with you the WIP. Would you love to contribute to this? Please let us to know. :smile:

**Comment by mli-max** [comment]
@jodosha I can do some work on this one

**Comment by mli-max** [comment]
@agenteo Are you working with that?

**Comment by jodosha** [comment]
@mli-max Please work on it. This is the branch: https://github.com/lotus/lotus/tree/cli-refactoring

The build is red :red_circle: , because it's WIP. I suspect this will be a long refactoring, take your time to make it right :+1:

**Comment by mli-max** [comment]
@jodosha thanks.

**Comment by agenteo** [comment]
@mli-max thanks for picking it up!

**Comment by mli-max** [comment]
This issue will be fixed with #297

**Comment by joneslee85** [comment]
Fixed
